src/css/global/_media-queries.scss

```diff
@@ -0,0 +1,34 @@
+/* Screen size mixins */
+
+/* Use this when the general mobile approach doesn't
+*  fit the screen
+*/
+@mixin tiny-only {
+    @media only screen and (max-width: 360px) {
+        @content;
+    }
+}
+
+@mixin mobile-only {
+    @media only screen and (max-width: $breakpoint-xs-max) {
+      @content;
+    }
+  }
+
+@mixin sm-and-up {
+    @media only screen and (min-width: $breakpoint-sm-min) {
+        @content;
+    }
+}
+
+@mixin md-and-up {
```

Comment:
this comment might be out of scope, let me know. but currently anywhere that uses a medium breakpoint actually needs to be switched to use small bp, it is a part of this ticket: https://github.com/telosnetwork/telos-wallet/issues/258

so actually instead of `sm-and`up` and `md-and`up` we actually just want a single mixin called `desktop-only` 

```scss
@media only screen and (min-width: $breakpoint-sm-min) {
  @content;
}
```

Reply:
this can be done here in this PR or added as a comment to that issue, doesn't matter which

Reply:
Yeah, I moved that issue back to ready to start once I started making the changes.

It kinda felt weird having the desktop menu on `md` size screens. I feel it took too much space from them, so I thought it would be best to be re-evaluated and addressed in a new PR which includes all decision-making and its reasons.

Would that be okay?